packages/client/src/components/interface/Header/Header.tsx

```diff
@@ -288,11 +307,38 @@ class HeaderComp extends React.Component<IProps, IState> {
     this.props.redirectToAuthentication()
   }
 
+  isLandingPage = () => {
+    const role = this.props.userDetails && this.props.userDetails.role
+    const location = this.props.history.location.pathname as any
+    if (
+      FIELD_AGENT_ROLES.includes(role as string) &&
+      [HOME].includes(location)
```

Comment:
Why do you need `[HOME]` as an `array`? And why the location is `any`. Can we define a type for that?

Reply:
I used an array to maintain if any role has multiple landing pages. But for the time being, I have changed it to string and also removed any type from the location variable as it automatically infers to string.